packages/monitors/test/ContractMonitor.js

```diff
@@ -473,7 +495,26 @@ contract("ContractMonitor.js", function(accounts) {
           errorThrown2 = true;
         }
         assert.isTrue(errorThrown2);
+
+        let errorThrown3;
```

Comment:
test to validate that it correctly process (and rejects) `empProps`. The same test is applied through the other tests as well.